src/operations/eval.ts

```diff
@@ -68,7 +68,7 @@ export class EvalOperation extends CommandOperation<Document> {
       }
 
       if (result) {
-        callback(MongoError.create({ message: `eval failed: ${result.errmsg}`, driver: true }));
+        callback(new MongoServerError({ message: `eval failed: ${result.errmsg}`, driver: true }));
```

Comment:
```suggestion
callback(new MongoServerError(`eval failed: ${result.errmsg}`));
```

The `driver: true` flag is legacy from when mongodb-core was a seperate package. We don't need to keep this.